playground/src/options.ts

```diff
@@ -0,0 +1,267 @@
+
+// This file is auto-generated by `cargo dev generate-playground-options`.
+export interface OptionGroup {
+    name: string;
+    fields: {
+        name: string;
+        default: string;
+        type: string;
+    }[];
+};
+
+export const AVAILABLE_OPTIONS: OptionGroup[] = [
```

Comment:
Can we omit the settings that don't make sense in a non-filesystem context? Like `src`, `exclude`, etc.

Reply:
Yes, good idea. Done.